data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -635,22 +635,22 @@ spec:
                       description: PowerVS is the configuration used when installing
                         on IBM Power VS.
                       properties:
-                        memory:
-                          description: Memory defines the memory in GB for the instance.
-                          type: string
+                        memoryGiB:
+                          description: memoryGiB is the size of a virtual machine's
+                            memory, in GiB.
+                          format: int32
+                          type: integer
                         procType:
                           description: ProcType defines the processor sharing model
-                            for the instance. Must be one of {capped, dedicated, shared}.
```

Comment:
is there a way we can leave in this enum? it was specifically requested

Reply:
this must be autogenerated from somewhere else, @Karthik-K-N can you the content from which this one got autogenerated.

Reply:
Thats because i removed it [here](https://github.com/openshift/installer/pull/5873/files#r881333983), as we have added validation in api with CamelCase: https://github.com/openshift/api/blob/4e43481ff9d3bcdbc2dee1504ecdc1cfa4ca745b/machine/v1/types_powervsprovider.go#L97